app/lib/bk/compat/parsers/bitrise/workflows.rb

```diff
@@ -14,20 +16,43 @@ def parse_workflow(wf_name, wf_config)
           step_key, step_config = step.first
           cmd_step << translate_step(step_key, step_config)
         end
-
+        process_workflow_envs(wf_config, cmd_step)
         cmd_step
       end
 
+      def process_workflow_envs(wf_config, cmd_step)
+        cmd_step.env ||= {}
+        BK::Compat::Bitrise::EnvHelper.process_env_variables(wf_config['envs'], cmd_step.env)
```

Comment:
if `BK::Compat::Bitrise::EnvHelper.process_env_variables` didn't take the second parameter and returned a dictionary (as mentioned in other comments) this should be:

```suggestion
        cmd_step.env.merge!(BK::Compat::Bitrise::EnvHelper.process_env_variables(wf_config['envs']))
```